op-node/sources/types_test.go

```diff
@@ -0,0 +1,37 @@
+		},
+		{
+			// TODO: JSON RPC providers are not yet returning the withdrawals root
```

Comment:
Alchemy Goerli node is returning withdrawals and withdrawalsRoot.  Mainnet won't yet have it as Shapella hasn't activated yet.